girderformindlogger/models/events.py

```diff
@@ -45,14 +45,18 @@ def upsertEvent(self, event, applet_id, event_id = None):
 
         if 'data' in event:
             newEvent['data'] = event['data']
-            if 'users' in event['data']:
+            if 'users' in event['data'] and isinstance(event['data']['users'], list):
                 newEvent['individualized'] = True
+
+                for i in range(0, len(event['data']['users'])):
```

Comment:
i think you can use list comprehension here

event['data']['users'] = [ObjectId(user) for user in event['data']['users']]